pytest-embedded/tests/test_base.py

```diff
@@ -165,7 +166,14 @@ def test_default_app_path(app):
 def test_parallel_run(parallel_count, parallel_index, res):
     from pytest_embedded.plugin import PytestEmbedded
 
-    fake_items = [1, 2, 3, 4, 5]
+    @dataclass
+    class _FakeItem:
```

Comment:
or we could use the `_pytest.nodes.Item` instead.

Reply:
Otherwise, we can't guarantee there won't be any breaking changes some day...

Reply:
According to documentation `Direct construction of _pytest.nodes.Item has been deprecated, please use _pytest.nodes.Item.from_parent.` >> [doc](https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent) .

I think we can leave it as it is since the `_pytest.nodes.Item.from_parent()` approach much larger than the current one

Reply:
ah, good to know. I agree.